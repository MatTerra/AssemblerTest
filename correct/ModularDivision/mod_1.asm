mod_1: begin
section data
result: space
section text
n: extern
mod_2: extern
public finish
public divide
input n
input n
jmp mod_2
divide: load n
div n
store result
output result
finish: stop
end
