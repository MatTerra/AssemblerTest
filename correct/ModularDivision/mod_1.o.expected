H: mod_1
H: 16
H: 0101010101010100
H: U mod_2 5
H: U n 1 3 7 9
H: D divide 6
H: D finish 14
T: 12 00 12 00 05 00 10 00 04 00 11 15 13 15 14 0
